src/tools/logger.ts

```diff
@@ -1,33 +1,34 @@
-const pino = require('pino');
-const config = require('config');
+import * as type from '../types';
+import pino from 'pino';
+import config from 'config';
 
 const logger = pino(config.get('logger'));
 
 let id = 0;
-const timers = {};
+const timers: type.ITimer = {};
 
 logger.startTimer = () => {
   const profiler = {
     timerId: `timer${id}`,
-    done(meta) {
+    done(meta: any) {
       const start = timers[profiler.timerId];
       if (!start) {
         return;
       }
-      const duration = new Date() - start;
+      const duration = +(new Date()) - start;
```

Comment:
```diff
- +(new Date())
+ Date.now()
```
From Airbnb JavaScript Style Guide:
> [22.3](https://github.com/airbnb/javascript#coercion--numbers) Numbers: Use `Number` for type casting and `parseInt` always with a radix for parsing strings. eslint: [`radix`](https://eslint.org/docs/rules/radix) [`no-new-wrappers`](https://eslint.org/docs/rules/no-new-wrappers)
>
> Why? The `parseInt` function produces an integer value dictated by interpretation of the contents of the string argument according to the specified radix. Leading whitespace in string is ignored. If radix is `undefined` or `0`, it is assumed to be `10` except when the number begins with the character pairs `0x` or `0X`, in which case a radix of 16 is assumed. This differs from ECMAScript 3, which merely discouraged (but allowed) octal interpretation. Many implementations have not adopted this behavior as of 2013. And, because older browsers must be supported, always specify a radix.
>
> ```javascript
> const inputValue = '4';
>
> // bad
> const val = new Number(inputValue);
>
> // bad
> const val = +inputValue;
>
> // bad
> const val = inputValue >> 0;
>
> // bad
> const val = parseInt(inputValue);
>
> // good
> const val = Number(inputValue);
>
> // good
> const val = parseInt(inputValue, 10);
> ```